libraries/AP_Baro/AP_Baro.cpp

```diff
@@ -367,7 +402,8 @@ void AP_Baro::update(void)
             if (is_zero(ground_pressure) || isnan(ground_pressure) || isinf(ground_pressure)) {
                 sensors[i].ground_pressure = sensors[i].pressure;
             }
-            float altitude = get_altitude_difference(sensors[i].ground_pressure, sensors[i].pressure);
+            float altitude = get_altitude_difference(sensors[i].ground_pressure, sensors[i].pressure) +
```

Comment:
Up at line 380 I added a LPF to _alt_offset a few months ago for when step-inputs are done by companion computer or GCS. The result of that is _alt_offset_active. Then at line 405 you add your "raw" offset then at 410 the existing code adds my LPF'd offset again. The offset is applied twice. The offset at line 405 should be removed.